libs/libplist/Makefile

```diff
@@ -52,7 +51,7 @@ define Package/libplistcxx
   $(call Package/libplist/Default)
   SECTION:=libs
   CATEGORY:=Libraries
-  DEPENDS:=+libplist +libstdcpp
+  DEPENDS:=+libplist $(CXX_DEPENDS)
```

Comment:
For most users a change in base C++ library -- needs run testing.

Reply:
Then I'd wait until it can actually be run tested -- if it's not used by much and no other user who can test the functionality cares, then what's the rush?

Reply:
I'm not sure @jow- @thess @dedeckeh @hnyman @yousong @blogic and other members are happy with a 'shove it in attitude', because ultimately they are the ones stuck with the breakage and having to fix problems that end up in a release.

Reply:
Not sure what you mean. Several of my pull requests have involved fixing build failures ( https://downloads.openwrt.org/snapshots/faillogs/ ), most of which have not been caused by me.

As for the uclibc++ change, there's a patch on patchwork currently that fixes crtmpserver and gptfdisk (have not submitted a pull request yet).

Reply:
I don't mean you're creating build failures; I mean that lack of run testing means that there may be more subtle failures that show up once users try to actually use the changes.  Compilation is not a guarantor of a successful result.